proto/documents/v1alpha/comments.proto

```diff
@@ -0,0 +1,128 @@
+syntax = "proto3";
+
+package com.mintter.documents.v1alpha;
+
+import "google/protobuf/empty.proto";
+import "documents/v1alpha/documents.proto";
+
+option go_package = "mintter/backend/genproto/documents/v1alpha;documents";
+
+// Comments service provides the way to add comments to publications.
+service Comments {
+  // Creates a new conversation about a particular selection in a document.
+  rpc CreateConversation(CreateConversationRequest) returns (Conversation);
+
+  // Adds a comment to a previously existing conversation.
+  rpc AddComment(AddCommentRequest) returns (Block);
+
+  // Deletes an existing conversation.
+  rpc DeleteConversation(DeleteConversationRequest) returns (google.protobuf.Empty);
```

Comment:
As previously commented, Have we thought about the implications of this?

Reply:
All deletes methods are currently unimplemented for that reason.